### PR TITLE
Split managed and static data sources and rewording

### LINKED ISSUE #2191: [Doom] Navigation refresh for Data sources

### PR DESCRIPTION
**Pull request by spolu**
Fix https://github.com/dust-tt/dust/issues/2191

Split managed and static data sources. Tweaks to the menu.

Screenshot from 2023-10-19 11-55-58
Screenshot from 2023-10-19 11-56-30

### CODE REVIEW

File: front/components/sparkle/navigation.tsx [hunk 1]
```diff
@@ -84,7 +86,7 @@ export const topNavigation = ({
     nav.push({
       id: "settings",
       label: "Admin",
-      icon: KeyIcon,
+      icon: Cog6ToothIcon,
```
Comment (fontanierh) [inline front/components/sparkle/navigation.tsx:89]: oups 😬

**Comment by Duncid** [comment]
Will make it available later today.
No problem to push as is.

File: front/components/sparkle/navigation.tsx (hunk 1 above)
Comment (spolu) [reply front/components/sparkle/navigation.tsx:89]: I think we have consensus :)

File: front/components/sparkle/navigation.tsx [hunk 2]
```diff
@@ -112,39 +114,50 @@ export const subNavigationAdmin = ({
 
   nav.push({
     id: "conversation",
-    label: "Conversations",
+    label: null,
```
Comment (fontanierh) [inline front/components/sparkle/navigation.tsx:117]: Why null here ?

File: front/pages/w/[wId]/builder/data-sources/static.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        <div className="flex flex-col gap-4 pb-4">
+          <PageHeader
+            title="Data Sources"
+            icon={DocumentDuplicateIcon}
+            description="Make data available to the workspace. Manage the data manually or by API."
+          />
+
+          <SectionHeader
+            title="Data Sources"
+            description="Make uploaded text files (doc, pdf, text, ...) available to Dust."
```
Comment (fontanierh) [inline front/pages/w/[wId]/builder/data-sources/static.tsx:98]: Mmmmh do we really support anything but raw text here ?

**Review by fontanierh — DISMISSED** [review]
LGTM

File: front/components/sparkle/navigation.tsx (hunk 1 above)
Comment (fontanierh) [reply front/components/sparkle/navigation.tsx:89]: Yes, but sorry for updating the header and not the side menu !

File: front/components/sparkle/navigation.tsx (hunk 2 above)
Comment (spolu) [reply front/components/sparkle/navigation.tsx:117]: To match the figma with no section title for these 3 ones :+1:

**Comment by gabhubert** [comment]
I think there's a little more work over time on this, but my suggestion to land a cut quickly is: 

## in the first page
- replace Platform integrations by "Connections". 
- in the page, copy would be:
Connections
Manage connections to your products and the real-time data feeds Dust has access to
- remove the second title
- this makes the concept of "connected data source" disappear in the product, I think that's fine when replace with "connections"

## In the second page
- keep Data sources
- Copy on the page: 
Data Sources
Make more documents (.txt, ,md, .pdf)accessible to this workspace. Manage data sources manually or [via API](https://docs.dust.tt/data-sources-overview).
- remove the second title + tagline

**Comment by spolu** [comment]
This would look like this. Feels a bit empty but works for me. @Duncid wdyt?

Screenshot from 2023-10-19 16-52-33
Screenshot from 2023-10-19 16-52-41

**Comment by spolu** [comment]
Especially when you don't have any data source :grimacing: 
Screenshot from 2023-10-19 16-55-10

**Comment by spolu** [comment]
Screenshot from 2023-10-19 17-05-21
Screenshot from 2023-10-19 17-06-27

**Comment by spolu** [comment]
Will merge as is and we can of course iterate as needed

**Pull request merged**